### ISSUE TITLE
I'd like to compile it from the nix.

### ISSUE THREAD

**Issue by PlumpMath** [knupfer/haskell-emacs#69]
g.g please
Thanks.

**Comment by Anton-Latukha** [comment]
Develop & maintain Nix configuration is an effort that requires continuity.

I'd also liked for a lot of things to have Nix infra, but I am not ready to provide it & support it, so because I am not ready to do & support it I do not raise such questions in that case. It is in my case, since I am able to develop Nix configuration & support Haskell-Nix infra if needed.

**Comment by PlumpMath** [comment]
You are absolutely right. I'm not prepared to handle this with Haskell and nix. The first and this greed was only in emacs, so I was asked. That's a good point from you.
Nevertheless, I'm drawn to it. 😄;;
